src/darwin/Framework/CHIP/MTRDeviceController+XPC.h

```diff
@@ -183,6 +184,15 @@ typedef void (^MTRValuesHandler)(id _Nullable values, NSError * _Nullable error)
                              attributeId:(NSNumber * _Nullable)attributeId
                               completion:(MTRValuesHandler)completion;
 
+/**
+ * Requests downloading some logs
+ */
+- (void)downloadLogWithController:(id _Nullable)controller
+                           nodeId:(uint64_t)nodeId
```

Comment:
Is this public API?  If so, it should use our normal public API naming conventions.  And yes, I know other things on here don't.

Also, if this is public API does it need availability annotations?

Reply:
Agree it needs newly available, but base device appears to always use these number conventions

Reply:
Let’s keep with what we have, as we want to slowly go away from base device

Reply:
>  but base device appears to always use these number conventions

I'm not sure what you mean here.....